src/main/jbake/content/blog/2016/04/28/new-packaging.adoc

```diff
@@ -0,0 +1,129 @@
+= New Hawkular packaging
+Heiko W. Rupp, Stefan Negrea
+2016-4-28
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, manageiq, components
+
+_ [ This blog post is was sent before to the hawkular-dev mailing list ] _
```

Comment:
...post is was sent...  + it looks strange on the page http://209.132.178.114:10167/blog/2016/04/28/new-packaging.html  Not sure what the `_ [ something ] _` should do in adoc, but it seems not to work for the website